### ISSUE TITLE
PSS/E parser support transformer angle offset of more than 60 degrees

### ISSUE THREAD

**Issue by hei06j** [lanl-ansi/PowerModels.jl#918]
PSSE is capable of solving cases with transformer angle offsets of more than 60 degrees, which are not supported in PowerModels. 

The attached raw file is extracted and reduced from an existing example case in PSSE, which solves in PSSE, but not in PowerModels.

[case33_3winding_transformer.raw.zip](https://github.com/user-attachments/files/15857663/case33_3winding_transformer.raw.zip)

**Comment by ccoffrin** [comment]
I don't think this was addressed by https://github.com/lanl-ansi/PowerModels.jl/pull/919, is that correct?

**Comment by hei06j** [comment]
that is correct.
